.prettierrc.json

```diff
@@ -3,5 +3,6 @@
   "plugins": ["prettier-plugin-organize-imports", "prettier-plugin-packagejson"],
   "printWidth": 120,
   "semi": false,
-  "singleQuote": true
+  "singleQuote": true,
+  "embeddedLanguageFormatting": "off"
```

Comment:
Does anything break if we keep embedded language formatting? I'm wondering why there are so many test snapshot changes. For consistency, I would prefer to not disable it, unless that results in even more changes.